src/app/lib/views/movie_detail.js

```diff
@@ -232,7 +226,7 @@
                 $('.sub-dropdown').addClass('open');
                 $('.sub-dropdown-arrow').addClass('down');
             }
-            var self = this;
+
```

Comment:
same remark here : if it's unused, then yes, it shouldn't be in the code. But it requires to be in its own separated commit too